e2e-tests/cypress/fixtures/account.js

```diff
@@ -11,7 +11,7 @@ const {
 const account = {
   [FIRST_NAME]: 'Joe',
   [LAST_NAME]: 'Bloggs',
-  [EMAIL]: 'mock@email.com',
+  [EMAIL]: 'Tony.Barnes@ukexportfinance.gov.uk',
```

Comment:
can we use mock emails in tests? @abhi-markan do you agree?

Reply:
Yes please, no PII only mock emails, however we can use `ukexportfinance.gov.uk` domain if you wish.

Reply:
We're waiting for `exporter1@ukexport...` emails to be setup from Risual which will take a while. In the meantime we need to use an email that's in the Notify whitelist - gone with this for now. Happy to change it just _needs_ to be in the Notify whitelist.

Reply:
@ttbarnes  can we pass this as an environment variable instead? - so that the username is not here?

Reply:
Thanks guys great stuff - created a new environment variable `GOV_NOTIFY_EMAIL_RECIPIENT`. 

We have to use an email that is on the GOV notify whitelist, which will be a specific ukexportfinance.gov.uk domain. But its anonymised now via environment variable 😄